ua-snap/iem-webapp
# Remove beta sticker

Closes #505 

To test: 

 - ensure no scrap code with the beta warning exists
 - check all nav paths to ensure not present
 - review the code + nod approvingly at the slightly-changed wording about downloading an image from charts
